browserstack.json

```diff
@@ -4,15 +4,31 @@
 	"test_framework": "qunit",
 	"test_path": [
 		"test/index.html",
+		"test/startError.html",
 		"test/logs.html",
-		"test/amd.html"
+		"test/amd.html",
+		"test/autostart.html"
 	],
 	"browsers": [
 		"chrome_previous",
 		"chrome_current",
 		"firefox_previous",
 		"firefox_current",
+		"opera_previous",
 		"opera_current",
+		"safari_5_1",
+		{
+			"browser": "safari",
+			"browser_version": "6.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "6.2"
```

Comment:
Thanks, I was just testing I'll still refine this PR. 

I needed to push some commits to check Travis-CI